lib/models/tts.dart

```diff
@@ -73,6 +74,19 @@ class TtsModel extends ChangeNotifier {
     notifyListeners();
   }
 
+  void mute(TwitchUserModel model) {
+    ttsHandler.mutedUsers.add(model);
+    notifyListeners();
+  }
+
+  void unmute(TwitchUserModel model) {
+    if (ttsHandler.mutedUsers.isNotEmpty &&
```

Comment:
`Set.remove` returns whether or not the element was removed: https://api.dart.dev/stable/2.10.4/dart-core/Set/remove.html

So this can be:

```
if (ttsHandler.mutedUsers.remove(model)) {
  notifyListeners();
}
```

Actually the same is true for `add` :) https://api.dart.dev/stable/2.10.4/dart-core/Set/add.html